api/docs/CMakeLists.txt

```diff
@@ -56,6 +56,11 @@ function (gendox_to_dox gendox_in gendox_out path master_list)
   file(GLOB_RECURSE dox_glob "${path}/*.dox" "${path}/*.dox.in")
```

Comment:
> which works for all but aarchxx since the file structure is different.

I don't understand this: isn't this the source code directory structure?  That's identical for all the GA workflow jobs, right?  How can it be different for the aarchxx* jobs?  Please edit this sentence to clarify.